proteus/tests/SWFlow/solitary_wave.py

```diff
@@ -181,5 +181,3 @@ def hbeta_DBC(X, flag):
                                               boundaryConditions=boundaryConditions,
                                               bathymetry=bathymetry_function,
                                               analyticalSolution=analytical_Solution)
-mySWFlowProblem.physical_parameters['LINEAR_FRICTION'] = 0
```

Comment:
Just checking this since it wasn't part of the santos test.

Reply:
This line wasn't doing anything and instead of waiting for my next PR I just added it to this one since it was a small change.